SPECS-EXTENDED/python-hwdata/python-hwdata.spec

```diff
@@ -9,15 +9,15 @@ Distribution:   Mariner
 
 Name:		python-hwdata
 Version:	2.3.7
-Release:	9%{?dist}
+Release:	10%{?dist}
 Summary:	Python bindings to hwdata package
 BuildArch:  noarch
 License:	GPLv2
 URL:		https://github.com/xsuchy/python-hwdata
 # git clone https://github.com/xsuchy/python-hwdata.git
 # cd python-hwdata
 # tito build --tgz
-Source0:	%{name}-%{version}.tar.gz
+Source0:	https://github.com/xsuchy/python-hwdata/archive/%{name}-%{version}-1.tar.gz
```

Comment:
We probably need to re-upload the file if the name changed. Is it supposed to change at all?